python/cudaq/kernel/ast_bridge.py

```diff
@@ -1804,6 +1808,9 @@ def bodyBuilder(iterVal):
                     f'Unknown attribute on quantum operation {node.func.value.id} ({node.func.attr}). {maybeProposeOpAttrFix(node.func.value.id, node.func.attr)}'
                 )
 
+            self.emitFatalError(
+                f"Invalid function call - '{node.func.value.id}' is unknown.")
```

Comment:
Would it be possible to add a test to see this in action?

Reply:
Sure thing. Test added.